synapsis/src/main/resources/templates/pages/login.html

```diff
@@ -1,20 +1,130 @@
 <!DOCTYPE html>
-<html xmlns:th="http://thymeleaf.org" xmlns:layout="https://www.ultraq.net.nz/thymeleaf/layout" layout:decorate="_layout">
-<div layout:fragment="content">
+<html xmlns:th="https://www.ultraq.net.nz/thymeleaf/layout">
+<head>
+    <meta charset="UTF-8">
+    <title>Synapsis</title>
+    <script src="https://cdn.tailwindcss.com"></script>
+    <script>
+        tailwind.config = {
+            theme: {
+                extend: {
+                    colors: {
+                        'primary': '#fa2742',
+                        'primary-variant': '#fc9ca8',
+                        'black': '#1d2228',
+                        'black-variant': '#3c4653',
+                        'grey': '#8f949b',
+                        'light': '#e1e2e2',
+                        'facebook-blue': '#1877F2',
+                        'true-black': '#000000'
+                    },
+                    fontFamily: {
+                        sans: ['Merriweather Sans', 'sans-serif'],
+                        serif: ['Catamaran', 'serif'],
+                    }
+                }
+
+            }
+        }
+    </script>
+    <link rel="stylesheet" href="/css/styles.css">
+</head>
+<div>
 
     <form th:action="@{/login}" method="post">
 
         <p th:if="${param.error}">Invalid login credentials.</p>
```

Comment:
This error message doesn't seem to be styled. Was this meant to be part of this PR or another one?

Reply:
I didn't have the chance to style everything, this was just meant to be an intial PR to get some of the login styling in